indexer/chalicelib/utils.py

```diff
@@ -20,6 +21,24 @@
 module_logger = logging.getLogger(indexer_name + ".indexer")
 
 
+def create_dss_client(dss_host):
```

Comment:
Why are these and other HCA changes part of this PR?

Reply:
I don't know, something it weird.